tests/cache/test-vary-cache.php

```diff
@@ -30,6 +30,7 @@ public function setUp(): void {
 	}
 
 	public function tearDown(): void {
+		Constant_Mocker::clear();
```

Comment:
We have to clear mocked constants in `tearDown()`; otherwise, it is possible that another test will break because another test has mocked an unexpected constant.